usage/src/main/java/com/cloud/usage/parser/NetworkUsageParser.java

```diff
@@ -80,7 +80,7 @@ public static boolean parse(AccountVO account, Date startDate, Date endDate) {
             long zoneId = usageNetwork.getZoneId();
             String key = "" + zoneId;
             if (usageNetwork.getHostId() != 0) {
-                key += "-Host" + usageNetwork.getHostId();
+                key += "-Host" + usageNetwork.getHostId() + "-Network-" + usageNetwork.getNetworkId();
```

Comment:
should networkid be checked for 0 as well?

Reply:
@DaanHoogland this was not needed for networkid 0. the issue was only for extra nics.